setup.py

```diff
@@ -106,7 +106,8 @@ def write_version(filename=os.path.join(*['airflow',
 ]
 celery = [
     'celery>=3.1.17',
-    'flower>=0.7.3'
+    'flower>=0.7.3',
+    'karellen-kombu-ext==4.0.2'
```

Comment:
While this is a temporary solution, we also need to make sure that the dependencies we introduce are maintained. I really hope that your PR against kombu gets approved.